src/serializer.js

```diff
@@ -440,12 +466,21 @@ var Serializer = (function () {
       case 'Variable':
         return expr.toNT()
       case 'Literal':
+        var val = expr.value.toString() // should be a string already
         if (expr.datatype && this.flags.indexOf('x') < 0) { // Supress native numbers
           switch (expr.datatype.uri) {
+
             case 'http://www.w3.org/2001/XMLSchema#integer':
-              return expr.value.toString()
+              return val
+
+            case 'http://www.w3.org/2001/XMLSchema#decimal': // In urtle must have dot
```

Comment:
IN structures.n3 there is the a decimal:     :count 16;  
(maybe would be good to add negative and zero and large tests too, but that tests the functin exists)